docs/guides/hooks-list.rst

```diff
@@ -454,6 +454,20 @@ Note that not all hooks apply to instant notifications.
 	 * ``headers`` - an array of headers
 	 * ``params`` - other parameters inherited from the notification object or passed directly to ``elgg_send_email()``
 
+**email:message, system**
+	Triggered by ``elgg_send_email()``.
+	Applies to **subscriptions** and **instant** notifications with ``email`` method.
+	This hook allows you to alter an instance of ``\Zend\Mail\Message`` before it is passed to the email transport.
+	
+	``$params`` contains:
+
+	 * ``to`` - email address or string in the form ``Name <name@example.org>`` of the recipient
+	 * ``from`` - email address or string in the form ``Name <name@example.org>`` of the sender
```

Comment:
If I wrote a handler I'd rather receive an Address object for these, but I guess these could be pulled from the Message.